### ISSUE TITLE
Improper default config for Docker

### ISSUE THREAD

**Issue by chevdor** [firefly-iii/firefly-iii#2532]
**Bug description**
I am a new user and wanted to get started. Seeing a Docker image, I checked the doc and saw I could get started with a single container (I thought...).

This lead to a first failure. 

Obviously, a DB is required. This is when I bumped into the docker-compose file. I thought great, I finally can test.

Here start the issues. The doc tries to guide the user but this is super confusing. The doc mentions that the default expectation is MySQL. Docker-compose however contains a postgresql container. The default passwords also don't match.

My suggestion would be to have an all in one docker-compose, maybe named `docker-compose-allinone.yml` that does not depend on any `.env` file and gets firefly up and running with good default values as defined by the devs. That would allow users to get a local instance running in no time.

Currently, despite following the doc, patching docker-compose to use mysql and settings up a .env file, all I get is:
![image](https://user-images.githubusercontent.com/738724/64154101-6af98400-ce30-11e9-937a-ccfecd75172d.png)

and logs that pretty much say nothing:
![image](https://user-images.githubusercontent.com/738724/64154130-79e03680-ce30-11e9-9199-36cde5124f46.png)

Disclaimer, I did not have much time to test and I have not experience with PHP.

**Steps to reproduce**
Follow the doc...

**Expected behavior**
Firefly runs

**Comment by JC5** [comment]
Thanks for letting me know. I'll do a thorough check of the documentation. A lot has changed in the past release.

I can't promise an "all-in-one" file, but the example env file AND the docker-compose file should work out of the box, as you say.

Thanks again, I hope to push some updates to the docs and Firefly III soon.

**Comment by chevdor** [comment]
Hey thanks for the lightspeed feedback!
I am really looking forward to having it running because what I have seen so far looks amazing!
From my initial tests, it looked like the issue was that the DB did not exist and was not created. I did not check in detail though.

**Comment by JC5** [comment]
Yep a lot of things are not really consistent. I also see the docker compose file refers to the `develop` tag, which should be the `latest` tag. Sorry about that.

I'm working through the docs right now, and I'm updating files as I go. I have some code to optimize so it'll be end of the week before I can push 4.8.1. But that should come with consistent and working Docker (compose) files.

**Comment by chevdor** [comment]
Hey no worries and thanks for picking it up so quick!
Let me know once you have something ready, I'd be more than happy to test and report my findings.

**Comment by JC5** [comment]
Try the new instructions:

http://docs.firefly-iii.org/en/latest/installation/docker.html

Instead of the files recommended, use the following two files:

https://raw.githubusercontent.com/firefly-iii/firefly-iii/develop/docker-compose.yml
https://raw.githubusercontent.com/firefly-iii/firefly-iii/develop/.env.example

**Comment by chevdor** [comment]
tldr: it works!

Here is what I did as a script:
```
mkdir -p /tmp/firefly
cd /tmp/firefly
wget https://raw.githubusercontent.com/firefly-iii/firefly-iii/develop/docker-compose.yml
wget https://raw.githubusercontent.com/firefly-iii/firefly-iii/develop/.env.example -O .env
docker-compose up
echo Now wait a few seconds and head at http://localhost:80/
```

**Comment by JC5** [comment]
:+1:
